addons/website_links/static/src/js/website_links.js

```diff
@@ -359,11 +467,14 @@ publicWidget.registry.websiteLinks = publicWidget.Widget.extend({
             if ('error' in result) {
                 // Handle errors
                 if (result.error === 'empty_url') {
-                    $('.notification').html('<div class="alert alert-danger">The URL is empty.</div>');
-                } else if (result.error === 'url_not_found') {
-                    $('.notification').html('<div class="alert alert-danger">URL not found (404)</div>');
+                    notificationElement.innerHTML =
+                        "<div class='alert alert-danger'>The URL is empty.</div>";
```

Comment:
I know previously it was like that but this will not translate the string ```The URL is empty.```, please use ```_t```

Check all other place for the same translation issue.

Reply:
ALright so for all the instances where we are adding text via innerHtml should i make changes with _t ?